### DISCUSSION TITLE
http://127.0.0.1:8000/ohif，There is no model to choose

### DISCUSSION THREAD

**Discussion by 865979216** [Project-MONAI/MONAILabel#1670] [Q&A]
![image](https://github.com/Project-MONAI/MONAILabel/assets/97671318/0541bc4b-5182-49fe-b52f-6a1cd8338b3c)
like this。

**Comment by 865979216** [comment]
![image](https://github.com/Project-MONAI/MONAILabel/assets/97671318/33ff107f-163e-4bc8-a467-e6f0636fbe7f)
monailabel start_server --app apps/radiology --studies http://127.0.0.1:8042/dicom-web --conf models segmentation

**Comment by 865979216** [comment]
![image](https://github.com/Project-MONAI/MONAILabel/assets/97671318/4c0b5031-fd0d-463b-8162-8a3818302789)

**Comment by 865979216** [comment]
![image](https://github.com/Project-MONAI/MONAILabel/assets/97671318/fb019cbd-b4ab-47c7-8993-7174ca782573)

**Comment by joeadham** [comment]
![image](https://github.com/Project-MONAI/MONAILabel/assets/81246343/0cfabd46-a28a-4a49-98f9-30554a3ee54a)

Try clicking on the refresh icon next to the server url and wait for a bit

**Reply by 865979216** [reply]
I clicked on the refresh icon, and the CMD displayed the following content
![image](https://github.com/Project-MONAI/MONAILabel/assets/97671318/924da7e2-6836-430c-a4f6-671897855db6)

**Comment by diazandr3s** [comment]
Hi @865979216,

Have you tried connecting to the MONAI Label server by clicking on the connect button? From the first image you've shared, it seems the OHIF isn't connected to the MONAI Label server. I can't see the training parameters and the model name.

Let us know,

**Reply by 865979216** [reply]
I clicked on the refresh icon, and the CMD displayed the following content
![image](https://github.com/Project-MONAI/MONAILabel/assets/97671318/2e340f78-7305-4cc1-ad69-677001d02c2d)

**Reply by diazandr3s** [reply]
How many images do you have loaded in Orthanc? Is this happening with the other cases?

**Reply by 865979216** [reply]
I have loaded MRA and CT images, each containing over 100 images。
![image](https://github.com/Project-MONAI/MONAILabel/assets/97671318/869759bb-c41b-4c61-8f8e-8b9a25927911)
I think it's the problem with the above pictures。

**Reply by 865979216** [reply]
Maybe not “Modality”， “Value”？。
![image](https://github.com/Project-MONAI/MONAILabel/assets/97671318/1b1c39bb-591c-49d5-8fb8-5dad1f3d04b0)
